src/org/usfirst/frc/team2706/robot/commands/GetTargets.java

```diff
@@ -0,0 +1,39 @@
+package org.usfirst.frc.team2706.robot.commands;
+
+import org.usfirst.frc.team2706.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ * A test command that uses the camera to get target information.
+ */
+public class GetTargets extends Command {
```

Comment:
OK. @BookSmart123 please take care of this and update the commit so we can get this on the robot :)